dump/dump-to-bucket.py

```diff
@@ -292,7 +292,7 @@ def main(verbose, book, host, raw_bucket, baked_bucket, resources_bucket, bucket
     if bucket:
         raw_bucket = baked_bucket = resources_bucket = bucket
         raw_prefix = 'raw/'
-        baked_prefix = 'baked/'
+        baked_prefix = 'contents/'
```

Comment:
Super minor: I noticed that in the docstring at the top of this file it also references baked content with expected structure which may be worth making a corresponding update to just to keep those comments aligned with the code.